app/controllers/search_controller.rb

```diff
@@ -1,4 +1,6 @@
 class SearchController < ApplicationController
+  before_action :require_pek_admin, only: %i[suggest_leader]
```

Comment:
```suggestion
  before_action :require_pek_admin, only: :suggest_leader
```
For one action you can pass an element instead of an array.